src/components/pages/ParentHome/ParentNavbar.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
```

Comment:
I think this can be removed? Although, connect specifically might be one of those that does not get used explicitly but still needs to be imported... Something to check!